src/Avalonia.Controls/WindowBase.cs

```diff
@@ -29,6 +29,12 @@ public class WindowBase : TopLevel
         public static readonly DirectProperty<WindowBase, bool> IsActiveProperty =
             AvaloniaProperty.RegisterDirect<WindowBase, bool>(nameof(IsActive), o => o.IsActive);
 
+        /// <summary>
+        /// Defines the <see cref="Owner"/> property.
+        /// </summary>
+        public static readonly StyledProperty<WindowBase> OwnerProperty =
```

Comment:
This probably can be a `DirectProperty` as I don't think it will need to be styled (in reality `DirectProperty`s _can_ be styled but their styling is limited - I think what they support will be good enough)